plot.py

```diff
@@ -0,0 +1,89 @@
+import matplotlib.image as mpimg
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.offsetbox import AnnotationBbox, OffsetImage, TextArea
+
+# data = np.load('peephole_benchmark_data_small.npz')
+#data = np.load("peephole_benchmark_data_geomspace25.npz")
+#data = np.load('timeit_peephole_benchmark_data.npz')
+data = np.load('timeit_peephole_benchmark_data_geom25.npz')
+
+loopsizes = data["loopsizes"] * 2  # each loop has 4 gates
+walltimes = data["walltimes"]
+cputimes = data["cputimes"]
+programsizes = data["programsizes"]
+core_PL_times = data["core_PL_times"]
+
+
+print(loopsizes, walltimes, cputimes, programsizes, core_PL_times)
+
+
+plt.figure(figsize=(15, 9))
```

Comment:
One thing we may want to consider when harmonizing our plot styles is to use a consistent `figsize`, since that will change the size of the text relative to the rest of the figure when pasting these plots into slides or a document, for example.

Reply:
Good point, I actually had this in mind (hence why I chose the same colors as your plots) but didn't think about the font!

Let's talk about it when the data for all three plots are locked down. It also depends on the specs of the final presentation, etc.

cc @josh146